src/Tools/dotnet-dump/dotnet-dump.csproj

```diff
@@ -23,54 +22,5 @@
-      <PackagePath>tools/$(TargetFramework)/any/win-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\sos.dll" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/win-x64</PackagePath>
-    </None>
-    <None Include="$(SOSNETCoreBinaries)" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/win-x86</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\Windows_NT.x86.$(Configuration)\sos.dll" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/win-x86</PackagePath>
-    </None>
-    <None Include="$(SOSNETCoreBinaries)" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\Linux.x64.$(Configuration)\libsosplugin.so" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\Linux.x64.$(Configuration)\libsos.so" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\Linux.x64.$(Configuration)\sosdocsunix.txt" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-x64</PackagePath>
-    </None>
-    <None Include="$(SOSNETCoreBinaries)" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-musl-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\Linux-musl.x64.$(Configuration)\libsosplugin.so" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-musl-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\Linux-musl.x64.$(Configuration)\libsos.so" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-musl-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\Linux-musl.x64.$(Configuration)\sosdocsunix.txt" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/linux-musl-x64</PackagePath>
-    </None>
-    <None Include="$(SOSNETCoreBinaries)" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/osx-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\OSX.x64.$(Configuration)\libsosplugin.dylib" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/osx-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\OSX.x64.$(Configuration)\libsos.dylib" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/osx-x64</PackagePath>
-    </None>
-    <None Include="$(ArtifactsBinDir)\OSX.x64.$(Configuration)\sosdocsunix.txt" Pack="true" Visible="false">
-      <PackagePath>tools/$(TargetFramework)/any/osx-x64</PackagePath>
-    </None>
-  </ItemGroup>
+  <Import Project="$(RepositoryEngineeringDir)\sos-packaging.proj" />
```

Comment:
Just nitty things, but can we rename it to props? It's not really a project by itself. Also, why in the engineering directory? This is really product logic, even though it's only used at build time.

Reply:
Put everything related to building in eng.  Maybe it isn't completely the right place to put it.

I can rename it to .props.